saleor/graphql/product/tests/test_variant.py

```diff
@@ -1360,6 +1375,91 @@ def test_create_variant_with_rich_text_attribute(
     created_webhook_mock.assert_called_once_with(product.variants.last())
 
 
+@patch("saleor.plugins.manager.PluginsManager.product_variant_created")
+@freeze_time(datetime(2020, 5, 5, 5, 5, 5, tzinfo=pytz.utc))
+def test_create_variant_with_date_attribute(
```

Comment:
Shouldn't be two separate tests? You are checking date and datetime attribute here.

Reply:
Well true but essentially it uses the same save method and same mode field. So I would say it doesnt bring anything.
But sure, I'll add it.

Reply:
I saw that some tests are split for DateTime and Date and some are not. I don't treat DateTime and Date as the same value so I believe we should have separate tests for them. Or tests with `pytest.parametrize`